server/app/uk/gov/ons/addressIndex/server/modules/AddressParserModule.scala

```diff
@@ -0,0 +1,18 @@
+package uk.gov.ons.addressIndex.server.modules
+
+import javax.inject.Singleton
+import uk.gov.ons.addressIndex.crfscala.CrfScala.CrfTokenResult
+import uk.gov.ons.addressIndex.parsers.AddressParser
+
+@Singleton
+class AddressParserModule {
```

Comment:
Right, but we don't mock the implementation of this module anywhere (as far as I know), so injecting this module is not justified right now.